### PR TITLE
#trivial Update README with the list of all available screenlets

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,38 @@
...
+This repository contains the Liferay Screens source code, in addition to several example projects that illustrate how Screens is used. For detailed instructions on developing mobile apps with Screens, see the Screens documentation for [Android](https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/android-apps-with-liferay-screens) and [iOS](https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/ios-apps-with-liferay-screens) on the Liferay Developer Network (LDN). 
 
 Brief descriptions of this repository's contents are listed here:
 
 - `android`: the Liferay Screens for Android codebase, View Sets, and example projects.
 - `ios`: the Liferay Screens for iOS codebase, Themes, and example projects.
 
+For now, the available screenlets are:
+
+* **Login Screenlet**: Signs users in to a Liferay instance.
```
Comment (jmnavarro) [inline README.md]: I miss the link to reference pages in devnet

Comment (dgarciasarai) [inline README.md]: What links should I add? Each screenlet in both platforms?
Or should I reference the list of screenlets in devnet for each platform?

Comment (jmnavarro) [inline README.md]: What do you think about something like this?

**Login Screenlet** [[iOS](https://dev.liferay.com/develop/reference/-/knowledge_base/7-0/loginscreenlet-for-android) / [Android](https://dev.liferay.com/develop/reference/-/knowledge_base/7-0/loginscreenlet-for-android)]: Signs users in to a Liferay instance.

**Comment by jmnavarro** [comment]
👍

**Pull request merged**